packages/lit-virtualizer-examples/package.json

```diff
@@ -32,6 +32,7 @@
     },
     "dependencies": {
         "@babel/runtime": "^7.9.6",
+        "@lit-labs/virtualizer": "^0.7.0-pre.2",
```

Comment:
Let's go ahead and update this to "^0.7.0", since that's the current version in `lit-virtualizer`'s package.json (not updating this earlier was an oversight on my part).

Reply:
Ah, yes!  Updated.